components/blocks/carousel.tsx

```diff
@@ -28,7 +28,10 @@ export const Carousel = ({ data }) => {
   };
 
   return (
-    <Section className="hidden md:flex" color={data.backgroundColor}>
+    <Section
+      className={`${data.showOnMobileDevices ? "" : "hidden"} md:flex`}
```

Comment:
Don’t we still want to display: flex on devices smaller than md?

Reply:
Yes - it works on mobile device as well (tested from iPhone X view)